sarracen/__init__.py

```diff
@@ -4,11 +4,13 @@
 from .readers.read_phantom import read_phantom
 from .readers.read_gradsph import read_gradsph
 
+from .writers.write_phantom import write_phantom
+
 from .sarracen_dataframe import SarracenDataFrame
 
 from .interpolate import interpolate_2d, interpolate_2d_line, interpolate_3d_proj, interpolate_3d_cross
 from .render import render, streamlines, arrowplot
 
 import sarracen.disc
 
-__version__ = "1.2.3"
+__version__ = "1.2.4"
```

Comment:
No need to change the version yet. I do that just before I make an official release.